audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+        "    features=cloud_speech.RecognitionFeatures(\n",
+        "        enable_automatic_punctuation=True,\n",
+        "    ),\n",
+        "    auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "r2TqksAqhAgD"
+      },
+      "source": [
+        "#### Define the real-time request configuration\n",
+        "\n",
+        "Next, you define the real-time request passing the configuration and the audio sample you want to transcribe. Again, you don't need to define a recognizer."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Nh55mSzXhAgD"
+      },
+      "outputs": [],
+      "source": [
+        "real_time_request = cloud_speech.RecognizeRequest(\n",
+        "    recognizer=f\"projects/{PROJECT_ID}/locations/{LOCATION}/recognizers/_\",\n",
+        "    config=real_time_config,\n",
+        "    content=short_audio_sample_bytes,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "817YXVBli0aY"
+      },
+      "source": [
+        "#### Run the real-time recognition request\n",
+        "\n",
+        "Finally you submit the real-time recognition request."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "rc0cBrVsi7UG"
+      },
+      "outputs": [],
+      "source": [
+        "response = client.recognize(request=real_time_request)\n",
+        "\n",
+        "real_time_recognize_results = parse_real_time_recognize_response(response)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "J2vpMSv7CZ_2"
+      },
+      "source": [
+        "And you use an helper function to visualize transcriptions and the associated streams."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ezH51rLH4CBR"
+      },
+      "outputs": [],
+      "source": [
+        "for transcription, _ in real_time_recognize_results:\n",
+        "    print_transcription(short_audio_sample_bytes, transcription)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5M-lIwRJ43EC"
+      },
+      "source": [
+        "### Perform batch speech recognition"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LJxhFSg848MO"
+      },
+      "source": [
+        "#### Define batch recognition configuration\n",
+        "\n",
+        "You start defining the batch recognition configuration."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0CEQUL5_5BT-"
+      },
+      "outputs": [],
+      "source": [
+        "batch_recognition_config = cloud_speech.RecognitionConfig(\n",
```

Comment:
Similar to before, I think you can reuse the same `RecognitionConfig` instead of defining all of the parameters again? Unless you have a specific reason for defining separate ones.